static/docs/command-reference/pipeline/show.md

```diff
@@ -4,6 +4,12 @@ Show [stages](/doc/command-reference/run) in a pipeline that lead to the
 specified stage. By default it lists
 [DVC-files](/doc/user-guide/dvc-file-format).
 
+This command's output is automatically pushed to `less` command line tool if
```

Comment:
Do we pass additional flags to make it work with wide outputs? Probably should mention that as well.